projectile.el

```diff
@@ -1230,7 +1230,8 @@ explicitly."
   (projectile-project-root (or dir default-directory)))
 
 (defun projectile-default-project-name (project-root)
-  "Default function used create project name to be displayed based on the value of PROJECT-ROOT."
+  "Default function used create project name to be displayed based on the
```

Comment:
Checkdoc wants the first line of the docstring to be a complete sentence. This way we're solving one problem, but introducing another one.

Reply:
There are numerous other checkdoc warnings as well. But I agree we should not make it worse. Some of the others ones are really easy to fix but I'll leave that out of this pull request.